### PR TITLE
README with heading, description and tool catalogue

### PR DESCRIPTION
**Pull request by florisvdg**

### CODE REVIEW

**Review by SimonBarendse — APPROVED** [review]
:+1: 

Let's wait with merging this until the `brew install secrethub/tools/cli` actually works. Let's merge that first.

**Comment by mackenbach** [comment]
You could also create a `develop` branch and merge to that. Then we can keep adding packages until they are released to `master`.

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
-# homebrew-tools
\ No newline at end of file
+<p align="center">
+  <img src="https://secrethub.io/img/secrethub-logo.svg" alt="SecretHub" width="400px"/>
+</p>
+<h1 align="center">
+  <i>Homebrew</i>
+</h1>
+
+[SecretHub](https://secrethub.io)'s [Homebrew tap](https://docs.brew.sh/Taps) listing all of our tools that can be installed using Homebrew.
```
Comment (mackenbach) [inline README.md]: Consider changing this description to include the default oneliner:

```
A [Homebrew tap](https://docs.brew.sh/Taps) listing all of the SecretHub tools that can be installed using Homebrew.

> [SecretHub](https://secrethub.io) is a developer tool to keep passwords and other secrets out of automation scripts.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
-# homebrew-tools
\ No newline at end of file
+<p align="center">
+  <img src="https://secrethub.io/img/secrethub-logo.svg" alt="SecretHub" width="400px"/>
```
Comment (mackenbach) [inline README.md]: Also, let's add a link to the image.

File: README.md [hunk 3]
```diff
@@ -1 +1,14 @@
...
+  <i>Homebrew</i>
+</h1>
+
+[SecretHub](https://secrethub.io)'s [Homebrew tap](https://docs.brew.sh/Taps) listing all of our tools that can be installed using Homebrew.
+
+## Available tools
+
+| Name | Installation command |
+|------|----------------------|
+| CLI | `brew install secrethub/tools/cli` |
```
Comment (mackenbach) [inline README.md]: Add also a column for the GitHub repo with a link.

**Review by mackenbach — COMMENTED** [review]
I have a few small improvements for this README

**Comment by mackenbach** [comment]
Also, we could add a `Getting help` section with a link to discord and support email.

**Comment by florisvdg** [comment]
Don't think a help section here would add much. If you run into installation problems and need help, the repo of the actual tool you want to install should help you further (with its issues, README, Discord link, wiki, etc), not the repo listing all of the Homebrew specs.

**Pull request merged**